projects/plugins/crm/tests/php/automation/tools/class-automation-faker.php

```diff
@@ -229,6 +243,24 @@ public function invoice_data() {
 		);
 	}
 
+	/**
+	 * Return data for a dummy quote
+	 * @return array
+	 */
+	public function quote_data() {
+		return array(
+			'id'   => 1,
+			'data' => array(
+				'id_override' => '1',
+				'title'       => '',
+				'status'      => 'Unpaid',
```

Comment:
```suggestion
```
Really Quotes don't have a stored status like invoices. It's like a runtime status.